packages/forms/src/elements/faux-input/FauxInput.tsx

```diff
@@ -26,6 +26,7 @@ const FauxInputComponent = forwardRef<HTMLDivElement, IFauxInputProps>(
       onFocus,
       readOnly,
       validation,
+      mediaLayout,
```

Comment:
`StyledFauxInput` [sets](https://github.com/zendeskgarden/react-components/blob/4c887bef39da96479253a597d423f4f1d8abade2/packages/dropdowns.legacy/src/styled/select/StyledFauxInput.ts#L17) the `mediaLayout` to `true` and extends `FauxInput`, a React FC that returns a styled-component. In this case, a transient prop wouldn't be forwarded.